.github/workflows/main.yaml

```diff
@@ -1,6 +1,27 @@
 name: CI
 on: [push, pull_request, workflow_dispatch]
 jobs:
+  python-smoketests-ci:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    container: fedorapython/fedora-python-tox:latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install absolufy
+        run: |
+             python3 -m pip install absolufy-imports
+
+      - name: Enforce relative imports in package
+        run: >
+             find fmn -name \*.py -a \! -path fmn/database/migrations/env.py -print0
+             | xargs -0 absolufy-imports --never
```

Comment:
- Alembic imports the `env.py` file and migration scripts in a way that they don’t know they are inside the `fmn` package, any relative imports outside of the `fmn/database/migrations` directory would fail. In `env.py` there’s a comment about it, I’ll add one here, too.
- The default mode of `absolufy-imports` is to enfore imports are absolute instead of relative, `--never` does the opposite.